hub.py

```diff
@@ -114,23 +114,26 @@ def __init__(self, cluster, spec):
 
         self.nfs_share_name = f'/export/home-01/homes/{self.spec["name"]}'
 
-    def get_generated_config(self):
+    def get_generated_config(self, auth_provider, proxy_secret_key):
         """
         Generate config automatically for each hub
 
         Some config should be automatically set for all hubs based on
         spec in hubs.yaml. We generate them here.
 
-        Shouldn't have anything secret here.
+        WARNING: CONTAINS SECRET VALUES!
         """
 
-        return {
+        proxy_secret = hmac.new(proxy_secret_key, self.spec['name'].encode(), hashlib.sha256).hexdigest()
+
+        generated_config = {
             'nfsPVC': {
```

Comment:
This will get overwritten by hubs.yaml for the ephemeral hub right?

Reply:
from ephemeral-hub's values.yaml, we'll set `nfsPVC.enabled` to false, which will ignore this config. However, it would be nice to not need that - filed https://github.com/2i2c-org/pilot-hubs/issues/126 to tackle that.